frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -300,48 +364,178 @@ function ReaderBookmark:onShowBookmark()
     end
 
     function bm_menu:onMenuHold(item)
+        -- reverse the bookmarks (because KOReader bookmarks are sorted in reverse of natural page order):
+        -- not sure whether this is always the case?
+        local bookmarks = {}
+        for i = #bookmark.bookmarks, 1, -1 do
+            local temp = bookmark.bookmarks[i]
+            -- we need this reference to the unreversed, original bookmarks for deleting or renaming them:
+            temp.index = i
+            table.insert(bookmarks, temp)
+        end
+        local current_bookmark = 1
+        local page = ""
+        for nr, bmk in ipairs(bookmarks) do
+            if item.index == bmk.index then
+                current_bookmark = nr
+                -- grep the "page" string and page number:
+                page = item.text:match("^([A-Za-z]+ [0-9]+)")
+                if page == nil then
+                    page = ""
+                end
+                break
+            end
+        end
+        local title = T(_("Bookmark details  -  %1/%2"), tostring(current_bookmark), tostring(#bookmarks))
+        if page ~= "" then
+            title = title .. "  -  " .. string.lower(page)
+        end
+        local first_button_row = {
+            {
+                text = _("Go to"),
+                callback = function()
+                    UIManager:close(self.textviewer)
+                    UIManager:close(bookmark.bookmark_menu)
+                    bookmark.ui.link:addCurrentLocationToStack()
+                    bookmark:gotoBookmark(item.page)
+                end,
+            }
+        }
+        if bookmark.search_value ~= "" then
+            table.insert(first_button_row, {
+                text = "<",
```

Comment:
Please use (here and on the 2nd row) the same arrow symbols we use elsewhere, and make it BiDi aware (test the UI with arabic, `KO_RTL=yes ./kodev run`), see:
https://github.com/koreader/koreader/blob/4e281f00edaae361dc4a2325cd8e9a0e5622b9f7/frontend/apps/reader/modules/readersearch.lua#L147-L155